src/autocomplete.css

```diff
@@ -29,6 +29,14 @@
   outline: 3px solid #ffbf47;
 }
 
+.autocomplete__input.autocomplete__input--show-all-values {
```

Comment:
@edwardhorsford I think a modifier class isn't meant to live on its own like that, so I don't think it's necessary, but yes you can do it.